documentation/kernel-semantics.tex

```diff
@@ -1632,7 +1667,7 @@ \subsubsection{Function Invocation}
             \cesktranswheresplit%
                 {\evalconf{\StaticGet{\membermeta}}{\env}{\strace}{\cstrace}{\cex}{\econt}}%
                 {\contconf{\econt}{\deref{\mainenv(\membermeta)}}}%
-                {\text{where $\membermeta$ is a static field and $\deref{\mainenv(\membermeta)} \neq \emptyset$}}
+                {\text{where $\membermeta$ is a static field and $\contains(\mainenv, \membermeta) = \true$}}
```

Comment:
As I understand it, `\contains` is only used in "where" clauses of transitions, i.e. the boolean values produced by that function aren't used along with values from semantic domains.  We may add a note to the definition of `\contains` stating that it will be used as a proposition device rather than a function that produces a value from the boolean semantic domain.  In this case, we may omit `= \true` (just omit it) and `= \false` (`... not \contains...`) wherever we use `\contains`.